### ISSUE TITLE
Unclear about the meaning and usage of splitChunks.maxAsyncRequest 

### ISSUE THREAD

**Issue by kwzm** [webpack/webpack#8664]
I read the official docs, but I don't understand, can you give me some examples, I am very confused about the maxAsyncRequests.

**Comment by sokra** [comment]
It limits the requests per `import()`.

**Comment by kwzm** [comment]
@sokra Thanks, I have one more question, does this rule include common css request ?

**Comment by sokra** [comment]
No, it only counts the number of chunks as requests. Divide it by two if you have CSS in your chunks. That's a known bug, but the exact number of requests is not super important anyway. You can usually increase the number at bit without performance issues.

**Comment by zhangZippo** [comment]
> It limits the requests per `import()`.

hello, I don't then per import() means,could you show me one example? thanks
